crypto/secp256k1/field.py

```diff
@@ -38,11 +40,11 @@
 
 # fieldPrimeWordZero is word zero of the secp256k1 prime in the
 # internal field representation.  It is used during negation.
-fieldPrimeWordZero = 0x3fffc2f
+fieldPrimeWordZero = 0x3FFFC2F
```

Comment:
Really, Black? This seems unnecessary.

Reply:
[Heh](https://black.readthedocs.io/en/stable/the_black_code_style.html#numeric-literals). It's part of the "any color you like" charm. :slightly_smiling_face: